### DISCUSSION TITLE
Why Vaex can deal with String so fast?

### DISCUSSION THREAD

**Discussion by EdisonZiXing** [vaexio/vaex#1712] [Q&A]
Hello everyone! I'm learning from the Internet that Vaex coud deal with String dtype effectively.
I wonder why and how Vaex could do this? Are there some kinds of optimizitation?
Really appreicate for the instructions and any kinds of documents!!!
Thx and looking forward to your replies!!!!!!!!

**Comment by JovanVeljanoski** [comment]
This is a good place to start:
https://towardsdatascience.com/vaex-a-dataframe-with-super-strings-789b92e8d861
